# HG changeset patch
# Parent 39123f891f8f51969097c7a9bbfb15db8ad11917
# User sbr
# Date 1346494326 -7200

Scroll the waypoint list with mouse cursor over it. (r24502)

diff -r 39123f891f8f -r 4267f8b8cdc9 src/rail_gui.cpp
--- a/src/rail_gui.cpp	Thu Aug 30 17:45:09 2012 +0000
+++ b/src/rail_gui.cpp	Sat Sep 01 12:12:06 2012 +0200
@@ -1775,7 +1775,7 @@
 	EndContainer(),
 	NWidget(NWID_HORIZONTAL),
 		NWidget(NWID_MATRIX, COLOUR_DARK_GREEN, WID_BRW_WAYPOINT_MATRIX), SetPIP(3, 2, 3), SetScrollbar(WID_BRW_SCROLL),
-			NWidget(WWT_PANEL, COLOUR_DARK_GREEN, WID_BRW_WAYPOINT), SetMinimalSize(66, 60), SetDataTip(0x0, STR_WAYPOINT_GRAPHICS_TOOLTIP), EndContainer(),
+			NWidget(WWT_PANEL, COLOUR_DARK_GREEN, WID_BRW_WAYPOINT), SetMinimalSize(66, 60), SetDataTip(0x0, STR_WAYPOINT_GRAPHICS_TOOLTIP), SetScrollbar(WID_BRW_SCROLL), EndContainer(),
 		EndContainer(),
 		NWidget(NWID_VERTICAL),
 			NWidget(NWID_VSCROLLBAR, COLOUR_DARK_GREEN, WID_BRW_SCROLL),
